apps/src/lib/kits/maker/PlaygroundComponents.js

```diff
@@ -244,22 +245,35 @@ function initializeThermometer(board) {
   });
 }
 
-function initializeAccelerometer(board) {
+function initializeAccelerometer(board, expressBoard) {
   const accelerometer = new five.Accelerometer({
     board,
     controller: PlaygroundIO.Accelerometer
   });
   accelerometer.start = function() {
     accelerometer.io.sysexCommand([CP_COMMAND, CP_ACCEL_STREAM_ON]);
   };
-  accelerometer.getOrientation = function(orientationType) {
+  accelerometer.getOrientation = function(
+    orientationType,
+    express = expressBoard
```

Comment:
What does this default param do?

Reply:
This default param was to overcome some of the scoping in this function. I was having a hard time trying to figure out ways to get the value of expressBoard into this function. Any suggestions for other strategies?

Reply:
Ah I see! I think I'd suggest setting `this. expressBoard` during the init phase, so we can check that here.

Reply:
I pushed a sample commit here, if you want to pull/merge/cherry-pick: 40a6d4a1a585698172c3428d34b4e0b19927d5f2.